pwndbg/stack.py

```diff
@@ -45,6 +45,11 @@ def find(address):
 def find_upper_stack_boundary(addr, max_pages=1024):
     addr = pwndbg.memory.page_align(int(addr))
 
+    # For bare metal, we can not get the stack size from stack layout and page fault.
+    # Return current page for a walkaround.
```

Comment:
Maybe:

```python
# We can't get the stack size from stack layout and page fault on bare metal mode,
# so we return current page as a walkaround.
```

Btw this might be added to some baremetal docs too.

Reply:
Fixed.